src/main/java/org/ambraproject/wombat/service/PeerReviewServiceImpl.java

```diff
@@ -174,7 +207,29 @@ String getReviewXml(Map<String, ?> metadata) throws IOException {
 
   String getContent(ContentKey contentKey) throws IOException {
     CloseableHttpResponse response = corpusContentApi.request(contentKey, ImmutableList.of());
-    String letterContent = EntityUtils.toString(response.getEntity(), "UTF-8");
-    return letterContent;
+    String content = EntityUtils.toString(response.getEntity(), "UTF-8");
+    return content;
+  }
+
+  /**
+   * Extract received date from article xml.
+   * @param xml article xml
+   * @return received date.
+   */
+  String getReceivedDate(String xml) throws IOException {
+
+    String receiveDateXml = XmlUtil.extractElement(xml, "date", "date-type", "received");
+
+    String receivedDateString = String.format("%s %s %s",
+      XmlUtil.extractText(XmlUtil.extractElement(receiveDateXml, "month")),
+      XmlUtil.extractText(XmlUtil.extractElement(receiveDateXml, "day")),
+      XmlUtil.extractText(XmlUtil.extractElement(receiveDateXml, "year")));
+
+    DateTimeFormatter parseFormatter   = DateTimeFormatter.ofPattern("M d yyyy");
+    DateTimeFormatter displayFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy");
```

Comment:
re: dev note 3: 
This is the only place where the date is received in parts.  We're pulling all other dates as literal strings from the review XML.  I would not be surprised if Tina has specified the date format in the instructions for Apex, but it would be a good idea to ask @ramzizaz and Katie, and also ask @samV-plos if she intended to specify a particular format or wants to conform with what we receive from Apex.

Reply:
I'm using microdata in the markup, to enhance machine-readability.  The value of the datetime attribute is the date specified using YYYY-MM-DD format:
`<time datetime="2011-04-01">04/01/11</time>`

Reply:
I've only been populating the nodes that humans will see.  To that end, @samV-plos  your prototype indicates a format of  `MMMM d, yyyy` - e.g. `October 4, 2018` 

I've asked Tina in the ticket to confirm that format we see in her sample XML is what we should expect in the sub-article XML:  `d MMM yyyy` - e.g. `4 Oct 2018`.

@samV-plos we should create a new ticket to populate the nodes that are not intended for visual consumption. I am certain that those nodes have not been implemented to your expectations.